PWGEM/PhotonMeson/TableProducer/skimmerGammaConversions.cxx

```diff
@@ -240,38 +240,39 @@ struct skimmerGammaConversions {
                   v0.pxneg(), v0.pyneg(), v0.pzneg(),
                   v0.v0cosPA(collision.posX(), collision.posY(), collision.posZ()), v0.dcaV0daughters()); // if v0legs is empty, lastIndex = -1.
 
-        fillTrackTable(v0, pos, true);
-        fillTrackTable(v0, ele, false);
+        fillTrackTable(v0, pos);
+        fillTrackTable(v0, ele);
```

Comment:
Why do we no longer need the charge information as a bool? Is it because we know if the index in the table is even it's positive and if it's odd it is negative?

Reply:
there is a variable track.sign() to get charge.

Reply:
Oh okay that makes sense :D